budcamp/Homework3/Homework3/MainMenu.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class MainMenu
+    {
+        public MainMenu() { }
```

Comment:
This constructor is empty and can be safely removed. It will be auto generated by the compiler anyway.